packages/wdio-appium-service/src/launcher.js

```diff
@@ -0,0 +1,120 @@
+import logger from '@wdio/logger'
+import fs from 'fs-extra'
```

Comment:
You can also import all the functions instead of the complete library.

Reply:
Fixed 👍